data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -247,6 +247,12 @@ spec:
                             should use Ultra SSD disks. The valid values are Enabled,
                             Disabled.
                           type: string
+                        acceleratedNetworking:
+                          description: AcceleratedNetworking specifies whether 
+                          to enable accelerated networking. Accelerated networking
+                          enables single root I/O virtualization (SR-IOV) to a VM,
+                          greatly improving its  networking performance.
+                          type: boolean
```

Comment:
When everything is set, this file will need to be re-generated.

Reply:
the pre-merge checking 

./openshift-install explain installconfig.compute.platform.azure.acceleratedNetworking
FATAL Error executing openshift-install: failed to load schema: yaml: line 253: could not find expected ':' 

./openshift-install version
./openshift-install 4.11.0-0.ci.test-2022-04-11-085335-ci-ln-5qj34it-latest
built from commit daf78dcc0db163383c408966468cc700d42ab3d8
release image registry.build01.ci.openshift.org/ci-ln-5qj34it/release@sha256:cd9f8cc97b040d7652b19eb4b25c3152639dfbfbe493e386111cd106537b0550

Reply:
whether we need add platform.azure.defaultMachinePlatform.acceleratedNetworking to define acceleratedNetworking of compute and controlPlane both ? @staebler

Reply:
> whether we need add platform.azure.defaultMachinePlatform.acceleratedNetworking to define acceleratedNetworking of compute and controlPlane both ?

Yes. We need to support a user setting a default in `defaultMachinePlatform`.

Reply:
test PASS with ./openshift-install 4.11.0-0.ci.test-2022-04-12-022813-ci-ln-626pzw2-latest
built from commit 383bd6940c8bf7c3c792c2847e1fe86a1ec4f1d3
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  AcceleratedNetworking specifies whether to enable accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, greatly improving its networking performance. eg. values: "Accelerated", "Basic"